pkg/gardenlet/apis/config/v1alpha1/defaults.go

```diff
@@ -480,3 +484,31 @@ func SetDefaults_Logging(obj *Logging) {
 		obj.Loki.Garden.Storage = &DefaultCentralLokiStorage
 	}
 }
+
+// SetDefaults_ETCDConfig sets defaults for the ETCD.
+func SetDefaults_ETCDConfig(obj *ETCDConfig) {
```

Comment:
Could you add unit test [here](https://github.com/gardener/gardener/blob/master/pkg/gardenlet/apis/config/v1alpha1/defaults_test.go) for this function?